overlord/snapstate/handlers.go

```diff
@@ -452,10 +452,28 @@ var (
 	mountPollInterval = 1 * time.Second
 )
 
+func otherSnapsLike(st *state.State, instanceName string) (bool, error) {
```

Comment:
I'm not super happy with the name but I can't think of anything that feels more catchy. I was wonderi if  `otherInstancesFor(st, instanceName)`, `hasOtherInstances(st, instanceName)` would be more descriptive. If not its probably fine but I would add a small docstring as the name does not give too much away.

Reply:
I actually like `hasOtherInstances()`.